cluster/src/main/scala/io/snappydata/impl/LeadImpl.scala

```diff
@@ -342,6 +354,23 @@ class LeadImpl extends ServerImpl with Lead with Logging {
   override def stopAllNetworkServers(): Unit = {
     // nothing to do as none of the net servers are allowed to start.
   }
+
+  /**
+   * This method is used to start the zeppelin interpreter thread.
+   * As discussed by default zeppelin interpreter will be enabled.User can disable it by
+   * setting "zeppelin.interpreter.enable" to false in leads conf file.User can also specify
+   * the port on which intrepreter should listen using  property zeppelin.interpreter.port
+   * @param bootProperties
+   */
+  private def checkAndStartZeppelinInterpreter(bootProperties: Properties): Unit = {
+    //As discussed ZeppelinRemoteInterpreter Server will be enabled by default.
+    if (bootProperties.getProperty(Constant.ENABLE_ZEPPELIN_INTERPRETER, "true").equals("true")) {
+      val port = bootProperties.getProperty(Constant.ZEPPELIN_INTERPRETER_PORT, "3768").toInt
+      val remoteInterpreterServer: SnappyRemoteInterpreterServer = new SnappyRemoteInterpreterServer(port)
```

Comment:
Rather than calling it SnappyRemoteInterpreterServer can't we call it only SnappyInterpreterServer. As for lead node its not remote. Only for Zeppelin its remote.